recipes/pdcurses/all/conanfile.py

```diff
@@ -1,149 +1,198 @@
-from conans import AutoToolsBuildEnvironment, ConanFile, tools
-from conans.errors import ConanException, ConanInvalidConfiguration
 import os
 import re
 
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration, ConanException
+from conan.tools.apple import is_apple_os
+from conan.tools.build import cross_building
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import chdir, copy, get, load, replace_in_file, rmdir, save, rename
+from conan.tools.gnu import Autotools, AutotoolsToolchain
+from conan.tools.layout import basic_layout
+from conan.tools.microsoft import is_msvc, msvc_runtime_flag, unix_path, NMakeToolchain, NMakeDeps
+
+required_conan_version = ">=1.53.0"
+
 
 class PDCursesConan(ConanFile):
     name = "pdcurses"
     description = "PDCurses - a curses library for environments that don't fit the termcap/terminfo model"
-    topics = ("conan", "pdcurses", "curses", "ncurses")
+    license = "LicenseRef-LICENSE"
```

Comment:
It's my understanding (But I'm not a lawyer) that this should be something like 
```suggestion
    license = "DocumentRef-README.md:LicenseRef-pdcurses"
```

Reply:
(Because afaik you want to link the original document where the license text comes from, not where it's put into in the CCI package) - and as there's no "Public Domain" license (I really wish people would properly license it under CC0 or something!) then we're not sure that the best license id to put there is - that's why I use the package name itself

Reply:
Going on a tangent: I understand the issue of "public domain" not being a valid and reliable license, but I'm not a huge fan of the SPDX word salad being used in its place either. I'm not against it either, but IMHO, a "Public Domain" would be a perfectly fine (non-)license identifier, which is about as useful as the DocumentRef/LicenseRef currently - it only provides a signal that you will need to look at the license in a case-by-case manner. Following the LicenseRef format is a bit cargo-culty and does not help much - it's not cross-referenced anywhere else and the specific license files can always found under the licenses folder. I would much rather the put effort into using the `x OR y` and `x AND y` notation to disambiguate the multiply-licensed projects, if I had to choose.

Anyway, sorry about the rant. What I really wanted to say is that the newer `DocumentRef-README.md:LicenseRef-pdcurses` should be included in the https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/conanfile_attributes.md#license-attribute docs as well. :smile: